editor/edit-post/sidebar/page-attributes/index.js

```diff
@@ -2,50 +2,71 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { PanelBody, PanelRow } from '@wordpress/components';
+import { PanelBody, PanelRow, withAPIData } from '@wordpress/components';
+import { compose } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { PageAttributes as PageAttributesForm, PageAttributesCheck } from '../../../components';
+import { PageAttributesCheck, PageAttributesOrder, PageAttributesParent } from '../../../components';
 import { toggleSidebarPanel } from '../../../store/actions';
-import { isEditorSidebarPanelOpened } from '../../../store/selectors';
+import { isEditorSidebarPanelOpened, getCurrentPostType } from '../../../store/selectors';
```

Comment:
I know it's not a rule written in stone and we don't do it consistently but we should avoid using selectors that are not "UI" related in the `edit-post` folder. That way we could separate the `edit-post` to its own module in the future. Here it's fine to use `isEditorSidebarPanelOpened` but maybe not `getCurrentPostType`. That's one of the reasons these *Check Component exist and are "connected"